### PR TITLE
some fixes

### PR DESCRIPTION
**Pull request by maslovakat**

### CODE REVIEW

File: front/src/common/redux/auth/auth.saga.js [hunk 1]
```diff
@@ -5,9 +5,9 @@ import * as types from "./types";
 function* loginSaga(payload) {
     try {
         const response = yield call(loginApi, payload);
-        
-        localStorage.setItem("token", response.data.token);
-        localStorage.setItem("user", JSON.stringify(response.data.user));
+        console.log(response)
```
Comment (Alan18081) [inline front/src/common/redux/auth/auth.saga.js]: Useless `console.log`

File: front/src/common/redux/groups/groups.action.js [hunk 2]
```diff
@@ -1,22 +1,24 @@
...
+import {
+    SET_CURRENT_GROUP, CHANGE_UPDATING_STATUS_CURRENT_GROUP, UPDATE_CURRENT_GROUP, LOAD_GROUPS_DATA,
+    START_LOAD_CURRENT_GROUP_DATA, DELETE_STUDENT_FROM_GROUP, CREATE_GROUP, CREATE_GROUP_SHOW_MODAL, CREATE_GROUP_HIDE_MODAL
+} from './types';
 
-export const createGroup = (data) => ({ 
+export const createGroup = (data) => ({
     type: CREATE_GROUP,
-    data 
+    data
```
Comment (Alan18081) [inline front/src/common/redux/groups/groups.action.js]: Please, use `payload` as a key

File: front/src/components/app/index.js [hunk 3]
```diff
@@ -22,6 +23,7 @@ class App extends Component {
                     <Route path="/forgotPassword/" component={ForgotPasswordPage} />
                     <Route path="/groups" component={GroupsPage} />
                     <Route path="/tasks" component={TasksList} />
+                    <Route path="/changePassword" component={ChangePasswordButtons} />
```
Comment (Alan18081) [inline front/src/components/app/index.js:26]: Why did you call it `ChangePasswordButtons`? I don't think that we need the word `Buttons` here

Comment (maslovakat) [reply front/src/components/app/index.js:26]: I use this component only for demonstrating modal window, it has only button

File: front/src/common/redux/groups/groups.action.js (hunk 2 above)
Comment (maslovakat) [reply front/src/common/redux/groups/groups.action.js]: done

File: front/src/common/redux/auth/auth.saga.js (hunk 1 above)
Comment (maslovakat) [reply front/src/common/redux/auth/auth.saga.js]: done

**Review by Alan18081 — APPROVED** [review]

**Pull request merged**